c_files/other/braces_issues.c

```diff
@@ -0,0 +1,7 @@
+int main() {
+    int x;
+    int y;
+    {
+    if(x > y){x = y;}
+}
```

Comment:
(same question about the outer braces)

Reply:
Cf. https://github.com/seiller/pymwp/issues/25 
As far as I know this is valid C code.

Reply:
CompCert says it is OK so must be :)